testing/matrix/linter/rules/roles/placement.go

```diff
@@ -157,6 +159,16 @@ func objectRBACPlacementServiceAccount(m utils.Module, object storage.StoreObjec
 			"Name of ServiceAccount should be equal to %q or %q",
 			serviceAccountName, expectedServiceAccountName,
 		)
+	case strings.HasSuffix(shortPath, "service-account.yaml"):
+		if namespace == ServiceAccountDefaultNs {
```

Comment:
It is supposed to be a namespace for user service accounts. To prevent overlapping and add our service account to this namespace, Deckhouse should always use a prefix, and it should be enforced.

Yet I still did not get why we needed to use this NS.

Reply:
It seemed to me that this is more like SA for end user and not for us. So this namespace looked like a proper place. Anyway, I fixed it.